### DISCUSSION TITLE
Enabling reverse sensor screen on R-Link 1

### DISCUSSION THREAD

**Discussion by ProfessorShroom** [cedricp/ddt4all#1107] [Q&A]
I'm trying to enable this display on R-Link 1, does anyone know where to look?
![239115882_4311272585625953_582585218229745720_n](https://user-images.githubusercontent.com/17574576/134241776-6b0c8b40-ade6-489f-8540-922b07a4c2a6.jpg)
I've managed to enable the rear camera screen and there's a space for the sensor display
![PXL_20210921_180719790](https://user-images.githubusercontent.com/17574576/134241998-0875c6d1-e284-4cad-a8c1-1f89e06d7fae.jpg)

**Comment by AndyIreland** [comment]
yep I think as i said before try here in the MFD settings : (although you do not seem to have Valeo UPA sensors show up in your list of ECUs on your car so that may be a problem 

![rear upa rlink](https://user-images.githubusercontent.com/49172217/134258380-c045b55b-ea12-4c85-816c-7e8214b30939.jpg)

**Comment by ProfessorShroom** [comment]
Yeah that enabled the reversing screen but not the sensors, does anyone else know?

**Comment by AndyIreland** [comment]
I should imagine the other cars that use the r-link (clio4 , Zoe, later Megane etc) have the parking aid computer in it and then talks to the r-link thats why?

**Comment by ProfessorShroom** [comment]
Do they? I got everything required retrofitted, the rear sensors work and you can disable them from the screen. Just don;t know how to enable that display

**Comment by AndyIreland** [comment]
Those segments only ever light up on the screen when an obstacle is behind the car ...... but you most probably know that already?

**Comment by ProfessorShroom** [comment]
Yes, but I want to enable that part, if you look at my photo you'll see the space is there but there's no car

**Comment by MellotronScratch** [comment]
You probably will need another PDC ecu. What's the partnumber of your current PDC ecu?

**Comment by ProfessorShroom** [comment]
No don't worry about that it's all there. I just need to find the option to enable it.

**Comment by AndyIreland** [comment]
> No don't worry about that it's all there. I just need to find the option to enable it.

if its all there then the UPA ECU should show up when you do a search with the magnifying glass in DDT4ALL - is it?

**Comment by ProfessorShroom** [comment]
Got it working. You do have to fit the newer parking module
![PXL_20211024_130725190](https://user-images.githubusercontent.com/17574576/138601228-905d4020-a9b9-4a66-862b-5fd42bfe4d47.jpg)

**Comment by catalup** [comment]
@charliehoward @AndyIreland @MellotronScratch 

Hello, I try to find a camera to mount on my Rlink-1. Can you help me with a link or with some information regarding this? The camera is 6v or 12v? Thank you!

**Comment by AndyIreland** [comment]
the output from the multi connector on the back of the rlink for the camera is 6v 200ma

![image](https://user-images.githubusercontent.com/49172217/196056951-91a76d39-8d2f-4a07-a252-0211815614d2.png)


![image](https://user-images.githubusercontent.com/49172217/196057023-c7f144e0-7ea7-46b2-8f26-f7d1c9168b95.png)

**Comment by HK-Zoe** [comment]
> Got it working. You do have to fit the newer parking module ![PXL_20211024_130725190](https://user-images.githubusercontent.com/17574576/138601228-905d4020-a9b9-4a66-862b-5fd42bfe4d47.jpg)

What settings did you edit to do it?

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/836 not bad!!

**Comment by Dorinmomo** [comment]
![image](https://github.com/cedricp/ddt4all/assets/170031235/3d6ddd54-d9a0-4b72-b67f-34bc1b48f085)
Hello. How did you make the image to work. I have a black screen also.....

**Comment by kajoQ** [comment]
Can somebody tell,im which settings can I turn off buzer beep sound PDC  in rlink - now I have enebled sound PDC in speakera and from buzzer.

**Reply by Renotech1** [reply]
Parking sensors?
You want only from buzzer or only from speakers?

**Reply by kajoQ** [reply]
I want only from speakers. Now I have from speakers and buzzers. And yes,parking sensora. I know i must do some changes in settings of pdc,radio and navigation - need know which.

**Reply by Renotech1** [reply]
![Screenshot_20251104_211738_Ecu Tweaker.jpg](https://github.com/user-attachments/assets/b35e2d4f-36ef-4a2f-b64e-0cf1ac27f3bb)

Maybe this ? Disable the buzzer . Always do reset to apply settings
